package.json

```diff
@@ -20,41 +20,42 @@
   },
   "homepage": "https://github.com/DFE-Digital/login.dfe.node-boilerplate#readme",
   "dependencies": {
-    "agentkeepalive": "^3.4.1",
-    "applicationinsights": "^1.0.3",
+    "agentkeepalive": "^4.5.0",
+    "applicationinsights": "^2.9.1",
     "body-parser": "^1.18.3",
     "email-validator": "^2.0.4",
     "express": "^4.16.3",
-    "express-validator": "^6.14.2",
-    "helmet": "^4.6.0",
-    "jsonwebtoken": "^8.3.0",
-    "login.dfe.dao": "^3.3.4",
+    "express-validator": "^7.0.1",
+    "helmet": "^7.1.0",
+    "jsonwebtoken": "^9.0.2",
     "lodash": "^4.17.15",
-    "login.dfe.api.auth": "github:DFE-Digital/login.dfe.api.auth#v2.0.2",
-    "login.dfe.audit.winston-sequelize-transport": "git+https://github.com/DFE-Digital/login.dfe.audit.winston-sequelize-transport.git#v2.13",
-    "login.dfe.express-error-handling": "git+https://github.com/DFE-Digital/login.dfe.express-error-handling.git#v1.2",
-    "login.dfe.healthcheck": "git+https://github.com/DFE-Digital/login.dfe.healthcheck.git#v1.8",
-    "login.dfe.jwt-strategies": "git+https://github.com/DFE-Digital/login.dfe.jwt-strategies.git#v3.0",
-    "login.dfe.public-api.jobs.client": "git+https://github.com/DFE-Digital/login.dfe.public-api.jobs.client.git#v2.0",
-    "login.dfe.request-promise-retry": "git+https://github.com/DFE-Digital/login.dfe.request-promise-retry.git#v2.1",
-    "login.dfe.sanitization": "git+https://github.com/DFE-Digital/login.dfe.sanitization.git#v2.0.0",
-    "login.dfe.winston-appinsights": "git+https://github.com/DFE-Digital/login.dfe.winston-appinsights.git#v3.0",
-    "niceware": "^1.0.6",
-    "promise-retry": "^1.1.1",
+    "login.dfe.api.auth": "github:DFE-Digital/login.dfe.api.auth#v2.3.0",
+    "login.dfe.audit.winston-sequelize-transport": "git+https://github.com/DFE-Digital/login.dfe.audit.winston-sequelize-transport.git#v3.1.0",
+    "login.dfe.dao": "4.0.0",
```

Comment:
can you please double check the DAO version? 
latest version is `login.dfe.dao 4.2.3` 
Is there any particular reason for removing the caret `^` prefix?